netsupportpolicy.md

```diff
@@ -0,0 +1,16 @@
+.NET support rules:
+1. ODL does not support .NET versions that are out-of-support
+2. .NET support has been tested prior to publishing a version with that .NET support
+3. removing support for a .NET version is a breaking change (and therefore requires a major version increment)
+4. adding support for a .NET version is not a breaking change
+5. ODL major versions are support for at least 1 year after release (TODO this claim was made in the last meeting, but i can't find in the existing doc)
+
+open items:
+1. How do we communicate the lifecycle moves for each ODL major version? (from John)
+    1. My proposal is to let the nuget release notes do this for us.
+2. How do we want to handle testing of .NET release candidates? (from many folks on the team)
+    1. My proposal is to test .NET support after the official .NET release, giving a 3 month runway.
+    2. Mike had feedback to use ODL beta releases, and had this comment: https://teams.microsoft.com/l/message/19:meeting_NWViNmQzN2ItMmQ0Mi00OGFmLWIxMzktZGVhY2EwMWMyMzBm@thread.v2/1720547775436?context=%7B%22contextType%22%3A%22chat%22%7D

```

Comment:
Should we establish a regular sync meeting with .net team? we should not, the .net team will give us a heads up whenever it seems applicable from their side

we **should** reach out to .NET team about this document regarding how to engage earlier in the process and see what might be established with other teams